### ISSUE TITLE
Narrator and JAWS screen reader are announcing invalid control type of "State" columnheader in detail list. 

### ISSUE THREAD

**Issue by anubhutisaini12** [microsoft/fluentui#16811]
Narrator and JAWS screen reader are announcing invalid control type of "State" columnheader in detail list. 

### Environment Information
Version: 2004 (OS Build: 20279.1)
Browser: Chrome Version 87.0.4280.88 (Official Build) (64-bit)
Screen reader: JAWS)

### Describe the issue:

Narrator and JAWS screen reader are announcing invalid control type of "State" columnheader in detail list. 

#### Actual behavior:

It is announcing "button" as control type for "State" columnheader while navigating on detail list

#### Expected behavior:

It should announce "columnheader" as control type for "State" columnheader while navigating.

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist

** Open narrator or any screen reader and navigate on to the columns heading in detail list.

**Comment by anubhutisaini12** [comment]
Any update for this issue ?

**Comment by joschect** [comment]
@anubhutisaini12 Sorry for the delay. I believe that we have a similar issue logged [here](https://github.com/microsoft/fluentui/issues/16788) but as I mention there I believe that button is a correct control type since it can be interacted with. I might be missing something so please let me know if that's not the case.
